Firmware/MotorControl/odrive_main.h

```diff
@@ -24,11 +24,7 @@ extern "C" {
 #include <cmsis_os.h>
 
 // Hardware configuration
-#if HW_VERSION_MAJOR == 3
-#include "board_config_v3.h"
```

Comment:
Please remove the now unused "board_config_v3.h" file as part of the merge request.